tools/osbuilder/rootfs-builder/fedora/Dockerfile.in

```diff
@@ -26,6 +26,7 @@ RUN dnf -y update && dnf install -y \
     libseccomp-devel \
     libstdc++-devel \
     libstdc++-static \
+    protobuf-compiler \
```

Comment:
hi @Amulyam24 , could you record somewhere (issue or commit or in code) why this package is needed? is it a ppc64le specific requirement?

Reply:
@wainersm, it's related to https://github.com/kata-containers/kata-containers/issues/1068

We better do the check for the architecture before installing.

Reply:
HI @wainersm, I have added the details about requiring the package in the issue as well as the commit.

Reply:
Thanks @Amulyam24 !